variables.tf

```diff
@@ -28,6 +28,12 @@ variable "wd_log_level" {
   default     = "info"
 }
 
+variable "enable_invocation_logs" {
+  description = "Turns on Waggledance invocation logs in log4j"
```

Comment:
NABD: Maybe worth to add
"Turns on Waggledance invocation logs in log4j, by default only "slow" 1 minute+ invocations are logged."

Reply:
👍 - done.